rebasehelper/tests/plugins/test_spec_hooks.py

```diff
@@ -33,12 +36,38 @@
 
 
 class TestSpecHook:
-    def test_typo_fix_spec_hook(self, spec_object):
-        assert '- This is chnagelog entry with some indentional typos' in spec_object.spec_content.section('%changelog')
-        TypoFix.run(spec_object, spec_object)
-        assert '- This is changelog entry with some intentional typos' in spec_object.spec_content.section('%changelog')
 
-    def test_paths_to_rpm_macros_spec_hook(self, spec_object):
+    @pytest.mark.parametrize('spec_attributes', [
+        {
+            'spec_content': '%changelog\n- This is chnagelog entry with some indentional typos'
+        }
+    ])
+    def test_typo_fix_spec_hook(self, mocked_spec_object):
+        assert '- This is chnagelog entry with some indentional typos' in \
+               mocked_spec_object.spec_content.section('%changelog')
+        TypoFix.run(mocked_spec_object, mocked_spec_object)
+        assert '- This is changelog entry with some intentional typos' in \
+               mocked_spec_object.spec_content.section('%changelog')
+
+    @pytest.mark.parametrize('spec_attributes', [
+        {
+            'spec_content': dedent("""\
+                %files
+                /usr/share/man/man1/*
+                /usr/bin/%{name}
+                %config(noreplace) /etc/test/test.conf
+
+                %files devel
+                %{_bindir}/test_example
+                %{_libdir}/my_test.so
+                /usr/share/test1.txt
+                /no/macros/here
+
+                """),
+            'macros': {},
```

Comment:
Why is this necessary?

Reply:
The hook requires `SpecFile.macros` to be set. `mocked_spec_content` only sets the macros if `macros` key is a part of `spec_attributes`. By setting it to empy dict, `SpecFile.macros` is set to `MacroHelper.dump()`